operations/system_management_health/Grafterm.md

```diff
@@ -0,0 +1,57 @@
+# Grafterm
+
+Visualize metrics dashboards on the terminal, like a simplified and minimalist version of **Grafana** for terminal.
+
+The utility(script) can be found in the /opt/cray/platform-utils directoty in all NCNs.
+
+## Running options
+
+Exit with `q` or `Esc`
+
+### Help
+
+```bash
+ncn-m001# ./grafterm.sh --help
+```
+
+### List available terminal dashboards
+
+```bash
+ncn-m001# ./grafterm.sh --list
+```
+
+### simple
+To view the dashboard, pass the value(dashboard json file) to the -c parameter to the script.
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json
+```
+
+### Relative time
+
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json -d 3h
+```
+
+### Refresh interval
+
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json -r 10s
+```
+
+```bash
+ncn-m001# ./grafterm.sh -c critical_services_dashboard.json  -d 3h -r 10s
+```
+
+### Fixed time
+
+Setting a fixed time range to visualize the metrics using duration notation. In this example is start at `now-23h` and end at `now-18h`
```

Comment:
- Change "setting" to "set"
- Change "In this example is start at `now-23h` and end at `now-18h`" to "In the following example, the start time is `now-23h` and the end time is `now-18h`."

Reply:
Done.